### PR TITLE
fix issue #941 bug

### PR DESCRIPTION
**Pull request by tuser4198**
## What is the purpose of the change

fix issue #941 , 把 start.sh的`127.0.0.1`改为'127.0.0.1'

## Brief changelog

dubbo/dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh

## Verifying this change

XXXX

Follow this checklist to help us incorporate your contribution quickly and easily:

- [x] Make sure there is a [GITHUB_issue](https://github.com/alibaba/dubbo/issues) filed for the change (usually before you start working on it). Trivial changes like typos do not require a GITHUB issue. Your pull request should address just this issue, without pulling in other changes - one PR resolves one issue. 
- [ ] Format the pull request title like `[Dubbo-XXX] Fix UnknownException when host config not exist`. Each commit in the pull request should have a meaningful subject line and body.
- [ ] Write a pull request description that is detailed enough to understand what the pull request does, how, and why.
- [ ] Write necessary unit-test to verify your logic correction, more mock a little better when cross module dependency exist. If the new feature or significant change is committed, please remember to add integration-test in [test module](https://github.com/alibaba/dubbo/tree/master/dubbo-test).
- [ ] Run `mvn clean install -DskipITs` to make sure unit-test pass. Run `mvn clean test-compile failsafe:integration-test`  to make sure integration-test pass.
- [ ] If this contribution is large, please file an [Apache Individual Contributor License Agreement](http://www.apache.org/licenses/#clas).

### CODE REVIEW

**Comment by tuser4198** [comment]
怎么解决CLA签订问题，我确认我今天早上已经签署过了，然后重新提pr还是提示未签署

File: dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh [hunk 1]
```diff
@@ -12,11 +12,11 @@ SERVER_PORT=`sed '/dubbo.protocol.port/!d;s/.*=//' conf/dubbo.properties | tr -d
 LOGS_FILE=`sed '/dubbo.log4j.file/!d;s/.*=//' conf/dubbo.properties | tr -d '\r'`
 
 if [ -z "$SERVER_HOST" ]; then
-    SERVER_HOST=`127.0.0.1`
+    SERVER_HOST='127.0.0.1'
 fi
 
 if [ -z "$SERVER_NAME" ]; then
-    SERVER_NAME=`hostname`
+    SERVER_NAME='hostname'
```
Comment (lujian0571) [inline dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh]: hostname是获取机器别名，需要加反引号"`"
加单引号"‘"后就变成字符串"hostname"了

如机器名为:"abc.local"
\`hostname\`的结果是:  abc.local
'hostname'的结果是: hostname

Comment (tuser4198) [reply dubbo-container/dubbo-container-api/src/main/resources/META-INF/assembly/bin/start.sh]: 嗯，谢谢提醒

**Comment by tuser4198** [comment]
由于账户原因，我重新提交下pr

**Pull request closed without merging**